playbooks/geniza_qa.yml

```diff
@@ -4,7 +4,7 @@
   become: true
   roles:
     - create_deployment
-    - build_dependencies
+    - cdh.common
```

Comment:
Common is a really generic name, can we please name it something more meaningful/descriptive? That tells me nothing about what it does.

Reply:
Should this be `cdh_common` here and throughout?